ingest/expression_files/mtx.py

```diff
@@ -97,13 +99,23 @@ def check_bundle(barcodes, genes, mtx_dimensions):
             raise ValueError(msg)
 
     @staticmethod
-    def check_duplicate_genes(gene_names: List):
-        unique_gene_names: List[str] = set(gene_names)
-        if len(unique_gene_names) != len(gene_names):
-            amount_of_duplicates = len(unique_gene_names) - len(gene_names)
+    def check_duplicates(names: List, file_type: str):
+        """Checks for duplicate values.
+        Barcode and gene files cannot contain duplicate values within the file
+
+        Parameters
+        ----------
+        names - Gene or cell values
+
+        file_type: Barcode or gene files. Used in error message
+        """
+        unique_names: List[str] = set(names)
+        if len(unique_names) != len(names):
+            amount_of_duplicates = abs(len(unique_names) - len(names))
```

Comment:
Clearer and shorter, given that `unique_names` will never be longer than `names`:

```suggestion
        if len(names) > len(unique_names):
            amount_of_duplicates = len(names) - len(unique_names)
```